scripts/build.py

```diff
@@ -100,13 +115,95 @@ def copy_files_source_to_target(source, target):
                 continue
             if source not in source_path:
                 continue
-            target_path = source_path.replace(
-                source, target)
+
+            target_path = source_path
+            relative_filepath = os.path.relpath(source_path, source)
+            if (not relative_filepath.endswith(FILE_EXTENSIONS_NOT_TO_RENAME) or
```

Comment:
I find this confusing; what if both conditions conflict? Is there a reason we can't just rename everything? What if new file types get introduced later on?

(Again, it might be worth adding comments to explain the answers to questions like these.)